src/Core/Services/Implementations/OrganizationService.cs

```diff
@@ -950,45 +950,25 @@ public async Task DisableTwoFactorProviderAsync(Organization organization, TwoFa
             await UpdateAsync(organization);
         }
 
-        public async Task<OrganizationUser> InviteUserAsync(Guid organizationId, Guid? invitingUserId, string email,
-            OrganizationUserType type, bool accessAll, string externalId, IEnumerable<SelectionReadOnly> collections)
-        {
-            var results = await InviteUserAsync(organizationId, invitingUserId, new List<string> { email },
-                type, accessAll, externalId, collections);
-            var result = results.FirstOrDefault();
-            if (result == null)
-            {
-                throw new BadRequestException("This user has already been invited.");
-            }
-            return result;
-        }
-
         public async Task<List<OrganizationUser>> InviteUserAsync(Guid organizationId, Guid? invitingUserId,
-            IEnumerable<string> emails, OrganizationUserType type, bool accessAll, string externalId,
-            IEnumerable<SelectionReadOnly> collections)
+            string externalId, OrganizationUserInvite invite)
         {
             var organization = await GetOrgById(organizationId);
             if (organization == null)
             {
                 throw new NotFoundException();
             }
 
```

Comment:
I would add some validation around `invite` to ensure it's not `null`, etc. There are assumptions made here, and perhaps they are the correct assumptions, but I'm not tracking upstream to see if being defensive here and validating not-null makes sense (let me know if it doesn't and that's cool with me).

Reply:
If invite is null or there aren't any emails being invited we the invite method wouldn't be able to do any inviting, so I resolved this by just returning Not Found if either of those things are null.